src/hotspot/share/gc/g1/g1CardSetContainers.hpp

```diff
@@ -26,15 +26,9 @@
 #define SHARE_GC_G1_G1CARDSETCONTAINERS_HPP
 
 #include "gc/g1/g1CardSet.hpp"
-#include "memory/allocation.hpp"
```

Comment:
allocation.hpp is used here - StackObj.  (We really ought to move that into it's own file, as was done for AllStatic.  It's a bit more complicated because of AllocatedObj.)  Presumably it's being picked up indirectly, but "include what you use".  (Hm, I thought that was in the style guide.  But not there yet - JDK-8252896.)

Reply:
Thanks @kimbarrett I missed that